static/js/teachers.ts

```diff
@@ -116,6 +116,48 @@ export function join_class(id: string, name: string) {
     });
 }
 
+export function invite_student(class_id: string) {
```

Comment:
We may want to re-run `build`, as this is currently failing locally for me with "invite_student" is not a function

Reply:
You can re-build locally using: `build-tools/heroku/generate-typescript --watch` (or without `--watch` if you just want to rebuild once)

Reply:
I see! I ran it locally just to be sure, but this generated changes - so I expect this to be part of this PR, or is this something that's the responsibility of the developer to always locally keep up to date (and changes in the bundle can be safely ignored)?

Would it be worth it to eventually add the bundles to .gitignore?

Reply:
The changes in the bundle can be safely ignored. They are automatically generated again on deployment. They do indeed cause merging conflicts, that's why your function call was failing. I merged main into this branch yesterday and discarded the app bundle files of this branch. 

We could add them to .gitignore. However, it is nice that you don't have to manually generate the files when reviewing a branch. Most times we only merge main into a branch (and discard the branches app bundle files) when we are ready to merge so the issue you encountered wouldn't occur.

Reply:
Yes they are not in the .gitignore by design, so people can easily check out the branch without the need for regenerating code (f.e. people working mainly in Python might not have the typescript stuff installed)